### DISCUSSION TITLE
Cannot open Draw.io file

### DISCUSSION THREAD

**Discussion by GerhardGenis** [jgraph/drawio-desktop#1855] [Q&A]
Trying to open the Draw.io file, the following message pops up: "Not a diagram file (error on line 1 at column 1: Start tag expected, '<' not found)" How can I fix it and retrieve the file

**Comment by davidjgraph** [comment]
attach it as a zip

**Reply by Jerseyno17** [reply]
[LATEST_REVISED_AM_20Nov24.drawio - Copy.zip](https://github.com/user-attachments/files/17845884/LATEST_REVISED_AM_20Nov24.drawio.-.Copy.zip)
@davidjgraph

**Reply by Jerseyno17** [reply]
same issue. unable to retrieve the file. 
@davidjgraph

**Reply by davidjgraph** [reply]
Unzip the file and look at it:

ˇÿˇ‡JFIFˇ‚ÿICC_PROFILE»0mntrRGB XYZ ‡acspˆ÷”-	desc$rXYZgXYZ(bXYZ<wtptPrTRCd(gTRCd(bTRCd(cprtå<mlucenUSsRGBXYZ o¢8ıêXYZ bô∑Ö⁄XYZ $†Ñ∂œXYZ ˆ÷”-paraffÚß
Y–
[mlucenUS Google Inc. 2016ˇ€C

That's not a drawio file.

**Comment by GerhardGenis** [comment]
Thx for the prompt response.

Kind regards

[A person in a suit  Description automatically generated]
Gerhard Genis
Independent Professional
EDP HCiI DiI MBA

Mobile: +27 (0) 83 274 1528
Standby Mobile: +27 (0) 61 529 6895
Email: ***@***.******@***.***>
Creation de la Uniq (Pty) Ltd
2023/124593/07

7 Bluebuck Street
The Hills Game Reserve Lifestyle Estate
Pretoria East
0059
South Africa

***@***.******@***.***
